docs/DOCS.md

```diff
@@ -0,0 +1,835 @@
+  the calendar and the datetime. This is one of the places where method signature is hard to remember, and
+  could perhaps be improved.
+
+- Class `CfDatetime`, which [composes](https://en.wikipedia.org/wiki/Composition_over_inheritance) a
+`CfTimeSystem` and a real-valued index to represent a specific time within a CF time system.
+   - Like `CalendarDatetime` (to which it is a parallel), `CfDatetime` offers only conversion methods
+   (e.g., `toISOString()`, `toCalendarDatetime`) and factories (e.g., `fromLooseFormat()`).
+   - Like `CalendarDatetime`, this is the class in which time arithmetic methods would be placed, but none are
+   currently needed, so none exist.
+
+## 6.5 Usage
+
+Playing at classes is all very well, till somebody loses their mind. How is this intended to be used?
+
+Here are some code snippets that show the application of these objects. Some make it obvious that it
+would be nicer to have (a) more consistent and/or flexible method signatures, and (b) more helper methods.
+
+### 6.5.1 `CalendarDatetime`: Datetimes in various calendars
+
+```javascript
+// Date in Gregorian calendar.
+const apollo11 = new CalendarDatetime(new GregorianCalendar(), 1969, 7, 20);
+console.log(apollo11.toISOString()); // -> 1969-07-20T00:00:00
+console.log(apollo11.toISOString(true)); // -> 1969-07-20
+
+// Or, using the pre-instantiated Gregorian calendar.
+const apollo11 = new CalendarDatetime(calendars.gregorian, 1969, 7, 20);
+
+// Dates in non-Gregorian calendar
+const endOfCentury = new CalendarDatetime(calendars['365_day'], 2099, 12, 31);
+console.log(endOfCentury.toLooseString()); // -> 2099/12/31 00:00:00
+console.log(endOfCentury.toLooseString(true)); // -> 2099/12/31
+```
+
+### 6.5.2 `CfTimeSystem`: CF time systems
+
+```javascript
+// A CF time system: days since 1950-01-01 in 360-day calendar; maximum time index 99999.
+const sinceDate = new CalendarDatetime(calendars['360_day'], 1950, 1, 1);
+const cfTimeSystem = new CfTimeSystem('days', sinceDate, 99999);
+```
+
+### 6.5.3 `CfDatetime`: CF time points
+
+```javascript
+// A CF datetime in the above time system, specified several different ways.
+const cfDatetime = new CfDatetime(cfTimeSystem, 720);
+const cfDatetime = CfDatetime.fromDatetime(cfTimeSystem, 1952, 1, 1);
+const cfDatetime = CfDatetime.fromLooseFormat(cfTimeSystem, '1952/01/01');
+console.log(cfDatetime.index); // -> 720
+console.log(cfDatetime.toLooseString(true)); // -> 1952/01/01
+```
+
+```javascript
+// Some possibly useful time points in the above time system.
+// Note these are `CfDatetime`s, not CalendarDatetime`s.
+const start = cfTimeSystem.firstCfDatetime();
+console.log(start.index);  // -> 0
+
+const end = cfTimeSystem.lastCfDatetime();
+console.log(end.index);  // -> 99999
+
+const today = cfTimeSystem.todayAsCfDatetime();
+console.log(today.index);  // -> some value around (<current year> - 1950) * 360
+```
```

Comment:
This file seems too long to me, I can easily imagine myself getting lost looking for some specific thing. I would suggest splitting it up my topic or task, and having an intro / table of contents / FAQ / whatever that explains what is in each smaller help file and links to it.